packages/accounts/src/msca/client.ts

```diff
@@ -38,6 +50,20 @@ export type CreateMultiOwnerModularAccountClientParams<
   "account"
 >;
 
+export type CreateMultisigModularAccountClientParams<
```

Comment:
nit: can we break these two client defs into 2 files within a `client` folder instead?

Reply:
+1, i think we'd also want to rename the modularAcount stuff to clarify its multiOwnerModularAccount right?